maps-compose/src/main/java/com/google/maps/android/compose/CameraPositionState.kt

```diff
@@ -104,7 +104,7 @@ class CameraPositionState(
 
     // The map currently associated with this CameraPositionState.
     // Guarded by `lock`.
-    private var map: GoogleMap? = null
+    var map: GoogleMap? = null
```

Comment:
Is there any reason to expose the GoogleMap instance there ?

Reply:
Same - was this intentional?